pages/learn/essentials/relations.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Relations
+layout: learn
+---
+
+# Relations
+
+Relations allow you to define relationships between tables, They are declared by using the built-in `field.relation()` declarative.
+
+To use the `field.relation()` declarative in your table, you must specify these props in the argument object:
+
+- `refereces`: Here you specify which table does this column relate to using the built-in `useTable()` declarative passing the table name as argument.
+- `relationship`: Here you specify the relationship type whether it is `one-to-one` or `many-to-one` according to your needs.
```

Comment:
it should say only

"One of 'many-to-one', 'one-to-one'"

These are known terminologies, they don't need details, just brief